src/Commands/Site/InfoCommand.php

```diff
@@ -27,6 +27,7 @@ class InfoCommand extends SiteCommand
      *     created: Created
      *     framework: Framework
      *     region: Region
+     *     region_label: Region
```

Comment:
@greg-1-anderson I experimented with how to call this second field. I tried "Region Code" for the GCP name vs. "Region Label" for the human-readable name but it just looked weird. I decided to call them both "Region" but I'm not too pleased about this either as it can create confusion when showing both of them (not sure why someone would do that but the option is there). What do you think?

Reply:
The user may select fields by their machine name (`region_label`) or their human-readable name (`Region`), so you shouldn't have two fields with exactly the same name. I think that "Region Code" is fine.